Sources/swift-parser-cli/Commands/PerformanceTest.swift

```diff
@@ -37,12 +41,40 @@ struct PerformanceTest: ParsableCommand {
       .filter { $0.pathExtension == "swift" }
       .map { try Data(contentsOf: $0) }
 
+    var fileTransition: [Data: (SourceFileSyntax, IncrementalParseNodeAffectRangeCollector)] = [:]
```

Comment:
Could you add labels to the tuple elements. It makes the access a lot easier to read IMO.